daprdocs/content/en/reference/components-reference/supported-pubsub/setup-azure-servicebus.md

```diff
@@ -149,6 +149,26 @@ To find out more details on the purpose of any of these metadata properties, ple
 
 > Note that all times are populated by the server and are not adjusted for clock skews.
 
+## Sending and receiving multiple messages
+
+Azure Service Bus natively supports sending and receiving multiple messages in a single operation. 
+
+### Configuring bulk publish
+
+To set the metadata for bulk publish operation, set the query parameters on the HTTP request or the gRPC metadata as documented [here]({{< ref pubsub_api >}})
+
+| Metadata | Default |
+|----------|---------|
+| `metadata.maxBulkPubBytes` | `131072` (128 KiB) |
+
+### Configuring bulk subscribe
+
+When subscribing to a topic, you can configure `bulkSubscribe` options. Refer to [Subscription methods]({{< ref subscription-methods >}}) for more details. (TODO: Add link to bulk subscribe docs)
```

Comment:
There is a TODO in this sentence

Reply:
Yes, TODO will be updated once #3077 is merged.